validator/accounts/accounts_list.go

```diff
@@ -134,7 +134,7 @@ func listDerivedKeymanagerAccounts(
 	au := aurora.NewAurora(true)
 	fmt.Printf("(keymanager kind) %s\n", au.BrightGreen("derived, (HD) hierarchical-deterministic").Bold())
 	fmt.Printf("(derivation format) %s\n", au.BrightGreen(keymanager.KeymanagerOpts().DerivedPathStructure).Bold())
-	validatingPubKeys, err := keymanager.FetchValidatingPublicKeys(ctx)
+	validatingPubKeys, err := keymanager.FetchAllValidatingPublicKeys(ctx)
```

Comment:
Would be helpful to list disabled keys differently here, perhaps in a follow-up PR @fabdarice

Reply:
https://github.com/prysmaticlabs/prysm/issues/7785#issue-741131221